### PR TITLE
Added user links to diary and achievements 

### PR DESCRIPTION
**Pull request by alcl2000**
Added user links to diary and achievements

### CODE REVIEW

**Review by KTC96 — APPROVED** [review]

File: mood_tracker/models.py [hunk 1]
```diff
@@ -24,15 +25,20 @@ class Achievements(models.Model):
...
+    def __str__(self):
+        return self.achievement_name
 
 
 class Diary(models.Model):
     title = models.CharField(max_length=200)
     content = models.TextField()
     date_created = models.DateTimeField(default=timezone.now)
-
+    owner = models.ForeignKey(User, related_name='diaries', blank=True, on_delete=models.CASCADE)
```
Comment (douglas86) [inline mood_tracker/models.py:38]: Have you updated your codebase as there were things that were just added to the model about 15 min ago.

Comment (alcl2000) [reply mood_tracker/models.py:38]: Yes, I edited the models

**Comment by douglas86** [comment]
class Diary(models.Model):
    title = models.CharField(max_length=200)
    content = models.TextField()
    mood = models.IntegerField()
    sleep = models.IntegerField()
    date_created = models.DateTimeField(default=timezone.now)

    def __str__(self):
        return self.title

    class Meta:
        verbose_name_plural = "Diaries"

**Comment by douglas86** [comment]
I was referring to mood and sleep that was added just before you created
the pr

**Pull request merged**